README.md

```diff
@@ -2,10 +2,10 @@
 
 The OCI Image Format project creates and maintains the software shipping container image format spec (OCI Image Format). The goal of this specification is to enable the creation of interoperable tools for building, transporting, and preparing a container image to run.
 
-This specification defines how to create an OCI Image, which will generally be done by a build system, and output an [image manifest](manifest.md), a set of [filesystem layers](layer.md), and an [image configuration](serialization.md).
-At a high level the image manifest contains metadata about the contents and dependencies of the image including the content-addressable identity of one or more filesystem serialization archives that will be unpacked to make up the final runnable filesystem.
+This specification defines how to create an OCI Image, which will generally be done by a build system, and output an [image manifest](manifest.md), a set of [filesystem layers](layer.md), and an [image configuration](config.md).
+At a high level the image manifest contains metadata about the contents and dependencies of the image including the content-addressable identity of one or more [filesystem layer changeset](layer.md) archives that will be unpacked to make up the final runnable filesystem.
 The image configuration includes information such as application arguments, environments, etc.
-The combination of the image manifest, image configuration, and one or more filesystem serializations is called the "OCI Image".
+The combination of the image manifest, image configuration, and one or more filesystem layers is called the "OCI Image".
```

Comment:
On Wed, Sep 14, 2016 at 02:29:31AM -0700, Antonio Murdaca wrote:
> >  The image configuration includes information such as application arguments, environments, etc.
> > -The combination of the image manifest, image configuration, and one or more filesystem serializations is called the "OCI Image".
> > +The combination of the image manifest, image configuration, and one or more filesystem layers is called the "OCI Image".
> 
> > Is there a reason not to make it OPTIONAL?  I think the empty array
> > (for “the rootfs is an empty directory”) is fine, and that both unset
> > and empty-array ‘layers’ get that idea across.
> 
> whatever, let's just move this conversation elsewhere not to clutter this PR anymore

Moved to #313 (with a bit in #308).